doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -240,6 +240,7 @@ Error list (HTTP response code in paranthesis):
 * LengthRequired (411). Zero/No Content-Length in PUT/POST/PATCH request
 * RequestEntityTooLarge (413). Payload too large
 * UnsupportedMediaType (415). Request content type is not supported.
+* InvalidModification (422). Request payload is missing some piece of information.
```

Comment:
Better would be to turn it around:

Some piece of information (is) missing in payload